.travis.yml

```diff
@@ -16,18 +16,18 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-23-a
+      env: SWIFT_SNAPSHOT=swift-4.2
```

Comment:
This line is not needed.  It'll default to the contents of _.swift-version_, in this case now to _4.2_.

Reply:
Sorry, I didn't see this the first time...